src/main/java/it/gov/pagopa/receipt/pdf/datastore/BizEventToReceipt.java

```diff
@@ -61,7 +61,7 @@ public void processBizEventToReceipt(
         List<Receipt> itemsDone = new ArrayList<>();
         Logger logger = context.getLogger();
 
-        String msg = String.format("BizEventEnrichment stat %s function - num events triggered %d", context.getInvocationId(), items.size());
+        String msg = String.format("BizEventToReceipt stat %s function - num events triggered %d", context.getInvocationId(), items.size());
```

Comment:
use [context](https://learn.microsoft.com/en-us/java/api/com.microsoft.azure.functions.executioncontext?view=azure-java-stable#method-details) to retrive correct fn name